common/src/stack/command/stack/commands/load/__init__.py

```diff
@@ -140,10 +140,24 @@ def load_controller(self, controllers, target=None):
 	def load_partition(self, partitions, target=None):
 		if not partitions:
 			return
-		
+
 		scope = self.get_scope()
 		assert not (scope != 'global' and target is None)
 
+		if scope == 'global':
+			cmd = 'add.storage.partition'
+		elif scope == 'appliance':
+			cmd = 'add.appliance.storage.partition'
+		elif scope == 'os':
+			cmd = 'add.os.storage.partition'
+		elif scope == 'host':
+			cmd = 'add.host.storage.partition'
+		else:
```

Comment:
Are environments currently unsupported?

Reply:
I copied that if/else from the `load storage partition` command, but it looks like there is a `add environment storage partition` command.